bin/powershell/check-windows-directory.ps1

```diff
@@ -0,0 +1,33 @@
+<#
+.SYNOPSIS 
+    Checks if  directory exist
+.DESCRIPTION
+    Checks if directory exist
+.Notes
+    FileName    : check-windows-directory.ps1
+    Author      : Patrice White - patrice.white@ge.com
+.LINK 
+    https://github.com/sensu-plugins/sensu-plugins-windows
+.PARAMETER LogName 
+    Required. The name of the directory.
+    Example -Dir C:\Users\dir
+.EXAMPLE
+    powershell.exe -file check-windows-directory.ps1 -Dir C:\Users\dir
+#>
+
+[CmdletBinding()]
+Param(
+  [Parameter(Mandatory=$True)]
+  [string]$Dir

```

Comment:
it would be nice if you could pass a comma separated list of directories to avoid needing to create multiple check definitions which would in turn instantiate the same check multiple times which adds overhead. This is by no means a requirement but if you are up for it I dont think it would be too hard to do though I really don't know powershell myself